### PR TITLE
[PATCH ArtworkDetails] Look update

### PR DESCRIPTION
**Pull request by oxaudo**
This is mostly bunch of cosmetic changes for ArtworkDetails. Brings the latest palette.
Will leave some inline comments here.

Latest look:
screen shot 2018-09-19 at 4 22 58 pm

### CODE REVIEW

File: src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAboutTheWorkFromPartner.tsx [hunk 1]
```diff
@@ -10,46 +18,70 @@ export interface ArtworkDetailsAboutTheWorkFromPartnerProps {
   artwork: ArtworkDetailsAboutTheWorkFromPartner_artwork
 }
 
-export const ArtworkDetailsAboutTheWorkFromPartner: React.SFC<
+const InitialsHolder = styled(Box)`
```
Comment (oxaudo) [inline src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAboutTheWorkFromPartner.tsx]: Not sure it was the best way to accomplish it - but this makes the avatar background when there is not image on partner profile.

File: src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsChecklist.tsx [hunk 2]
```diff
@@ -1,6 +1,5 @@
-import { Box, Sans, StackableBorderBox } from "@artsy/palette"
+import { Box, color, Sans, StackableBorderBox } from "@artsy/palette"
```
Comment (oxaudo) [inline src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsChecklist.tsx]: I think using color from Palette is a preferred way now, right?

File: src/Apps/Artwork/Components/ArtworkDetails/index.tsx [hunk 3]
```diff
@@ -132,7 +134,15 @@ export const ArtworkDetailsQueryRenderer = ({
                   />
                 )
               } else {
-                return null
+                return (
+                  <SpinnerContainer
```
Comment (oxaudo) [inline src/Apps/Artwork/Components/ArtworkDetails/index.tsx]: This adds a spinner while data for component loads

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 4df740174735e06fe1ea8248aa73c9e4aae19886

https://deploy-preview-1261--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/Banner.tsx [hunk 4]
```diff
@@ -48,6 +48,6 @@ export const SmallBanner = props => (
         {props.subHeadline}
       </Serif>
     </Flex>
-    <Avatar size="small" src={props.src} ml={2} />
+    <Avatar size="sm" src={props.src} ml={2} />
```
Comment (oxaudo) [inline src/Apps/Artwork/Components/Banner.tsx]: Update in Palette based on updated Avatar sizes there.

File: src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsChecklist.tsx (hunk 2 above)
Comment (damassi) [reply src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsChecklist.tsx]: Yupp

**Comment by damassi** [comment]
Everything looks super solid 👍

**Pull request merged**